src/main/kotlin/dk/cachet/carp/webservices/protocol/dto/GetLatestProtocolOverviewResponseDto.kt

```diff
@@ -0,0 +1,28 @@
+package dk.cachet.carp.webservices.protocol.dto
+
+import dk.cachet.carp.protocols.application.StudyProtocolSnapshot
+import java.time.Instant
+
+data class GetLatestProtocolOverviewResponseDto(
+    val ownerName: String?,
+    val snapshot: StudyProtocolSnapshot,
+    val lastVersionCreatedDate: Instant
+) {
+    /**
+     * The [from] function converts
+     * a data type [GetLatestProtocolResponseDto] to a [GetLatestProtocolOverviewResponseDto].
+     *
+     * @return A [GetLatestProtocolOverviewResponseDto] object containing the protocol from [snapshot].
+     */
+    companion object {
+        fun from(dto: GetLatestProtocolResponseDto): GetLatestProtocolOverviewResponseDto {
+            val snapshotOwnerName = dto.snapshot.name
```

Comment:
Uhm, is there any reason this is extracted into a variable, afaics you can just pass it in as the others